source/Calamari.Aws/Deployment/Conventions/UploadAwsS3Convention.cs

```diff
@@ -90,14 +89,13 @@ public void Install(RunningDeployment deployment)
                     {
                         if (response.IsSuccess())
                         {
-                            Log.Info(
-                                $"Saving variable \"Octopus.Action[{deployment.Variables["Octopus.Action.Name"]}].Output.Files[{response.BucketKey}]\"");
+                            Log.Info($"Saving variable \"Octopus.Action[{deployment.Variables["Octopus.Action.Name"]}].Output.Files[{response.BucketKey}]\"");
                             Log.SetOutputVariable($"Files[{response.BucketKey}]", response.Version);
                         }
                     }
                 });
             }
-            catch (AmazonS3Exception exception)
+             catch (AmazonS3Exception exception)
```

Comment:
This additional space _may_ cause the internet to explode

Reply:
I think it may already have if my internet's behaviour from yesterday is anything to go by! 😆